composer.json

```diff
@@ -48,6 +48,9 @@
         "sebastian/version": "^6.0.0",
         "staabm/side-effects-detector": "^1.0.5"
     },
+    "require-dev": {
+        "ext-pcntl": "*"
+    },
```

Comment:
`composer.lock` needs to be updated after `composer.json` is changed. This is the reason why the CI pipeline failed.